roles/engine-api-init/tasks/main.yml

```diff
@@ -1,86 +1,7 @@
 ---
-- name: Create default wazo-auth database preconfiguration
-  tempfile:
-    state: file
-    prefix: wazo_auth_debconf_file
-  register: wazo_auth_debconf_file
-
-- name: Copy wazo-auth database preconfiguration if needed
-  template:
-    src: templates/wazo-auth.cfg.j2
-    dest: "{{ wazo_auth_debconf_file.path }}"
-  when:
-    - engine_api_db_admin_password is defined
-    - engine_api_db_admin_user is defined
-    - engine_api_db_auth_name is defined
-    - engine_api_db_auth_password is defined
-    - engine_api_db_auth_user is defined
-    - engine_api_db_confd_name is defined
-    - engine_api_db_confd_password is defined
-    - engine_api_db_confd_user is defined
-    - engine_api_db_host is defined
-    - engine_api_db_port is defined
-
-- name: Reset database options for wazo-auth package
-  shell: "echo PURGE | debconf-communicate wazo-auth"
-
-- name: Reconfigure database options for wazo-auth package
-  command: debconf-set-selections "{{ wazo_auth_debconf_file.path }}"
-
-- name: Initialize wazo-auth database
-  command: dpkg-reconfigure wazo-auth --frontend noninteractive
-
-- name: Install wazo-auth-keys
-  apt:
-    name: wazo-auth-keys
-    state: latest
-
-- name: Create service users with wazo-auth-keys
-  command: wazo-auth-keys service update
-
-- name: Create default main database preconfiguration
-  tempfile:
-    state: file
-    prefix: xivo_managedb_debconf_file
-  register: xivo_managedb_debconf_file
-
-- name: Copy main database preconfiguration
-  template:
-    src: templates/xivo-manage-db.cfg.j2
-    dest: "{{ xivo_managedb_debconf_file.path }}"
-  when:
-    - engine_api_db_admin_password is defined
-    - engine_api_db_admin_user is defined
-    - engine_api_db_confd_name is defined
-    - engine_api_db_confd_password is defined
-    - engine_api_db_confd_user is defined
-    - engine_api_db_host is defined
-    - engine_api_db_port is defined
-
-- name: Reset main database options for wazo-auth package
-  shell: "echo PURGE | debconf-communicate xivo-manage-db"
-
-- name: Reconfigure main database options
-  command: debconf-set-selections "{{ xivo_managedb_debconf_file.path }}"
-
-- name: Initialize main database
-  command: dpkg-reconfigure xivo-manage-db --frontend noninteractive
-
-- name: Fix Twisted dropin.cache
-  command: twistd --help-reactors
-
-- name: Start Wazo services
-  service:
-    name: "{{ item }}"
-    state: restarted
-  loop:
-    - wazo-agid
-    - wazo-amid
-    - wazo-confd
-    - wazo-confgend
-    - wazo-calld
-    - wazo-provd
-    - wazo-websocketd
+- name: Install Wazo auth
+  include_role:
+    name: wazo_auth
```

Comment:
We should state on conventions for the naming of roles. We now have roles with word separator `-` and other roles with word separator `_` in their names. In other parts of Wazo, we preferentially use `-`. What do you think?